### ISSUE TITLE
Write tests

### ISSUE THREAD

**Issue by TylerGubala** [TylerGubala/blenderpy#34]
This repository has no tests written for it. I am not tremendously savvy with writing software tests. If someone could make a branch that allows tests to be run, that would be huge. Otherwise, I have to figure out what the best way to write tests is. CI would be best.

**Comment by caniko** [comment]
Do you want to test each function in `bpy`? That is a huge undertaking, and would be a huge boost for Blender in general.

Any funding? Isn't this beyond the scope of the repository?

**Comment by TylerGubala** [comment]
> Do you want to test each function in bpy?

No. Just a sane subset of tests, like "can I render?" small ones

> That is a huge undertaking

The whole thing is, that is the point. The more people get exposed to this codebase, maybe one of the people forking this repo will help.

> Any funding

None yet, not really the point, TBH

> Isn't this beyond the scope of the repository?

I think it's fine as long as the tests stay focused on "reasons why people want to use `bpy`"

Can I render stills and animation? Can I import/export a model in the same format (roundload)? Can I import in one format, and export in another (conversion)? Can I assign keyframes?

Those are the tests I was thinking of and it would not take long but I would prefer at least one other person get familiarized with `bpy` in principal in doing these tests because there is a lot to learn about the Blender build chain and most of my issues come from users installing it for the first time from PYPI, not from people who have created a fork.
